### DISCUSSION TITLE
Please make a major release bump if it contains breaking changes

### DISCUSSION THREAD

**Discussion by ChristianSch** [JLLeitschuh/ktlint-gradle#656] [General]
We let dependabot update from 11.0.0 to 11.3.1 and it broke our CI processes. And apparently yours as well: https://github.com/JLLeitschuh/ktlint-gradle/commit/83382192c0c3d3dede7b7ab323c1e74f5f51a99b

I would like you to consider bumping the major release version if things break after updating. Cheers

**Comment by JLLeitschuh** [comment]
Thanks. I personally don't like coupling our API's version changes to `ktlint`. Our API didn't change. The behaviour of the library we call, however, did. IMHO, if this is a concern, it should be the responsibility of the maintainer to lock the `ktlint` version to a specific version, which can be done via `version.set("VERSION")`. Additionally, you can configure the ktlint gradle more granularity using Gradle's version constraints by configuring the `ktlint` configuration
